.github/workflows/renovate.yml

```diff
@@ -40,7 +42,13 @@ jobs:
               "0 1 * * *" { "filter=workleap/ov-*" >> $env:GITHUB_OUTPUT }
               "0 2 * * *" { "filter=workleap/sg-*" >> $env:GITHUB_OUTPUT }
               "0 3 * * *" { "filter=workleap/wlp-*" >> $env:GITHUB_OUTPUT }
-              "0 4 * * *" { "filter=workleap/!(ov-*|sg-*|wlp-*)" >> $env:GITHUB_OUTPUT }
+              "0 4 * * *" { "filter=workleap/wl-[a-h]*" >> $env:GITHUB_OUTPUT }
```

Comment:
I think the regex is wrong if the intention is to split repo names by the letter they start with, it should be something like `wl-[a-h].*` or else it will only match repos with letters a-h in their name.